src/TC/inc/MemoryInterfaces.hpp

```diff
@@ -39,6 +42,8 @@ enum Pixel_Format {
   YUV422 = 11,
   P10 = 12,
   P12 = 13,
+  YUV444_10bit = 14,
+  NV12_Planar = 15
```

Comment:
NV12_Planar is only different from regular NV12 in the sense that `Y` and `UV` interleaved planes can be separated?

Reply:
Isn't NV12 Planar just YUV420 ?

Reply:
Ok i see it's 420 but with plane 1 and 2 guaranteed to be a continuous buffer.

Reply:
no, it's just plain NV12 but with chroma (UV interleaved) and luma (Y only) in potentially different locations (though I doubt that this will happen in practice).

Reply:
tracking this in new feature branch, marking as resolved for now